fastify4ts/utils/controllerFactory.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { routes, utils } from '@contrast/test-bench-utils';
+import { forEach } from 'lodash';
+import { FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+
+/*
+ * Custom response functions allow you to change the functionality or return
+ * value of a sink endpoint.
+ *
+ * @callback ResponseFn
+ * @param {*}                result  return value of the sink method
+ * @param {fastify.Request}  request fastify request object
+ * @param {fastify.Response} reply   fastify reply object
+ */
+
+/*
+ * @type {ResponseFn}
+ */
+
+const defaultRespond = (result: unknown, request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
These comments are useless if not attached to anything.

Since this is a ts file, they should be implemented as interfaces or types:

```suggestion
/**
 * Custom response functions allow you to change the functionality or return
 * value of a sink endpoint.
 */
interface RespondFn {
  (result: unknown, request: FastifyRequest, reply: FastifyReply): void;
}

const defaultRespond: RespondFn = (result, request, reply) => {
```